docs/recipes/how_to_contribute.qmd

```diff
@@ -74,3 +75,37 @@ git commit -m "your commit"
 ```bash
 git push origin branch_name
 ```
+
+## Step 3: Create a Pull Request: 
+Navigate to the DASCore repository on GitHub, and you should see a notification about your recent push. Click the "Compare & pull request" button to create a new pull request.
+
+
+# DASDAE users
+
+If you'd like to contribute to DASCore as a user, you should first fork the DASCore repository.Forking a repository allows you to freely experiment with changes without affecting the original project. Below is a detailed guide on how to do this.
+
+## Step 1: Fork the repositoy
+Go to the [DASCore repository](https://github.com/DASDAE/dascore) an click on "Fork".
+
+## Step 2: Clone your fork
+Once you have forked the repository, you need to clone it to your local machine to start making changes. Then navigate to your local repository:
+```bash
+cd dascore
+```
+
+## Step 3: Set Upstream Repository: 
+Add the original DASCore repository as an upstream repository, which will be useful for keeping your fork up to date with the original project:
+
+```bash
+git remote add upstream https://github.com/DASDAE/dascore.git
+```
+
+Verify that the new remote URL has been added:
+
+```bash
+git remote -v
+```
+
+You should see the original repository as `upstream` and your fork as `origin`.
+
+Then, create a new branch and follow steps 1 to 3 of the DASDAE developers section above.
```

Comment:
Probably want to add a bit about the pull request process from the fork. If they don't have write access to dasdae/dascore they need to push their new branch back to origin, then open a pull request from there.

Adding a link to [this](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) github docs could also be useful.